### PR TITLE
People picker in tab

### PR DESCRIPTION
**Pull request by NikitaJain-MSFT**

### CODE REVIEW

File: samples/tab-people-picker/nodejs/Manifest/manifest.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.12/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.12",
+  "version": "1.0.1",
```
Comment (v-chetsh) [inline samples/tab-people-picker/nodejs/Manifest/manifest.json:4]: 1.0.0

File: samples/tab-people-picker/nodejs/Manifest/manifest.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    {
+		"entityId": "peoplepickertab",
+		"name": "Tab",
+		"contentUrl": "https://{{base-url}}/tab",
+		"websiteUrl": "https://{{base-url}}/tab",
+		"scopes": [
+			"personal"
+		]
+    }
+  ],
```
Comment (v-chetsh) [inline samples/tab-people-picker/nodejs/Manifest/manifest.json:46]: Looks like alignment is not correct.

File: samples/tab-people-picker/nodejs/Readme.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+description: This is an tab app which shows the feature of client sdk people picker.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+contentType: samples
+createdDate: "06-10-2021 01:48:56"
```
Comment (v-chetsh) [inline samples/tab-people-picker/nodejs/Readme.md:12]: This date is wrong.

File: samples/tab-people-picker/nodejs/views/configure.ejs [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+</head>
+<body>
+    <div class="font-semibold font-title" id="textConfiguration"></div>
+<p>
+  <div>
+      Please select save to add the tab.
+  </div>
+</p>
+<script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
+<script src="https://statics.teams.cdn.office.net/sdk/v1.10.0/js/MicrosoftTeams.min.js" integrity="sha384-6oUzHUqESdbT3hNPDDZUa/OunUj5SoxuMXNek1Dwe6AmChzqc6EJhjVrJ93DY/Bv" crossorigin="anonymous"></script>
```
Comment (v-chetsh) [inline samples/tab-people-picker/nodejs/views/configure.ejs:16]: Let's put these on top.

File: samples/tab-people-picker/nodejs/views/index.ejs [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> Graph activity feed notification  Node js</title>
+</head>
+
+<body>
+    <div>
+        <h4>
+            Graph activity feed notification node js sample
```
Comment (v-chetsh) [inline samples/tab-people-picker/nodejs/views/index.ejs:14]: This is not Graph activity feed sample

**Review by v-chetsh — COMMENTED** [review]
Reviewed and provided my feedback.

File: README.md [hunk 6]
```diff
@@ -64,6 +64,7 @@ The [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDev
 |15|Create Conversational tab     | Microsoft Teams tab sample app for demonstrating create conversation tab | [View][tab-conversation#csharp]          |           |[View][tab-conversation#nodejs]
 |16| Product inspection | Demonstrating a feature where user can scan a product and mark it as approved/rejected. |[View][tab-product-inspection#csharp]|[View][productinspection#nodejs]|
 |17| Staggered Permission | This sample demos to get staggered graph api permissions. |[View][tab-staggered-permission#csharp]|[View][tab-staggered-permission#nodejs]|
+|18| Tab people picker | This is an tab app which shows the feature of client sdk people picker. ||[View][tab-people-picker#nodejs]|
```
Comment (AbhijeetThakur-msft) [inline README.md:67]: Let's rename this as : This sample shows tab capability with ...

File: samples/tab-people-picker/nodejs/views/tab.ejs [hunk 7]
```diff
@@ -0,0 +1,177 @@
+<head>
+    <script src="https://statics.teams.cdn.office.net/sdk/v1.11.0/js/MicrosoftTeams.min.js"
+        integrity="sha384-SCVF3m7OvDKnfAilUzYn2yozEvBeP8n/Oq0yTH8VUI70J4AzrqR70jzjdQ6DI8s2"
+        crossorigin="anonymous"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <meta charset="UTF-8">
+    <meta http-equiv="Content-Security-Policy"
+        content="default-src *; style-src 'self' 'unsafe-inline' http://localhost:3978; script-src 'self' 'unsafe-inline' 'unsafe-eval'">
```
Comment (AbhijeetThakur-msft) [inline samples/tab-people-picker/nodejs/views/tab.ejs:8]: Let's check for this static url.

**Review by AbhijeetThakur-msft — CHANGES_REQUESTED** [review]

**Review by AbhijeetThakur-msft — APPROVED** [review]

**Review by v-chetsh — APPROVED** [review]
Changes looks fine, approving it.

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**